### ISSUE TITLE
Unit 2 Overview

### ISSUE THREAD

**Issue by alex-pakalniskis** [IndexerDAO/docs#14]
@Idalithb

could you use ChatGPT to draft an overview paragraph for Unit 2? (similar type of content to the Unit 1 overview paragraph I asked you to revise).

Here's the unit content:
| Unit | Lesson  # | Topic(s) |
| --- | --- | --- |
| 2 | 1 | Intro to indexer operations Wallets (hot vs cold, indexer vs operator), Multisigs, Faucets, Network Registration |
| 2 | 2 | Getting started with containers Docker, docker-compose, Kubernetes (briefly), Helm (briefly) |
| 2 | 3 | Lab 2: Configure your indexing stack Deploy The Graph indexing stack and connect to your Gnosis RPC node |

**Comment by idalithb** [comment]
Yes! I will write/edit an overview and start going through the unit lessons next week, so let me know if there's additional documents you would like me to get done before the end of next week 7/14. I will be available to write/edit anything that IndexerDAO needs :)

**Comment by idalithb** [comment]
@alex-pakalniskis Update: currently working on a polished copy of this unit, but will have it ready by the end of this week :)

**Comment by idalithb** [comment]
@alex-pakalniskis I couldn't help myself and heavily edited the weird blob ChatGPT spit out, but Unit 2 is updated now :)
